pypeit/par/pypeitpar.py

```diff
@@ -77,6 +71,7 @@ def __init__(self, existing_par=None, foo=None):
 
 from pypeit.par.parset import ParSet
 from pypeit.par import util
+from pypeit.core.framematch import FrameTypeBitMask
```

Comment:
This whole module needs some TLC.  There are individual parameters and whole parameter sets that aren't used. We can push this to a different PR but it needs to be done at some point.

The uncommented parameters that are no longer used are a particular problem because they get documented.  We don't want a user changing a parameter and expect it to do something when it's never actually used.